### PR TITLE
Fix expected result in Custom Metrics - Stackdriver Adapter e2e test

### PR DESCRIPTION
**Pull request by kawych**
**What this PR does / why we need it**:
This PR fixes a bug in e2e tests for Custom Metrics - Stackdriver Adapter

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/instrumentation/monitoring/custom_metrics_stackdriver.go [hunk 1]
```diff
@@ -141,14 +141,11 @@ func testAdapter(f *framework.Framework, kubeClient clientset.Interface, customM
 	if err != nil {
```
Comment (fgrzadkowski) [inline test/e2e/instrumentation/monitoring/custom_metrics_stackdriver.go]: I can't comment on lines outside of this change so adding here:

I believe in line 124 you should be checking that you actually receive both metrics. Now you only check that you don't get extra metrics.

Comment (kawych) [reply test/e2e/instrumentation/monitoring/custom_metrics_stackdriver.go]: Done.

**Pull request merged**